### PR TITLE
NMS-12568: Add SystemExcecuteMonitor example into docs

### PR DESCRIPTION
**Pull request by mfuhrmann**
I got a hint from @indigo423 that should be also included in the docs. I've added an example for that.

https://issues.opennms.org/browse/NMS-12564

### CODE REVIEW

**Comment by indigo423** [comment]
We should improve the docs for Meridian releases as well, I've backported this and changed the target base to foundation-2018.

**Comment by indigo423** [comment]
We give the users two options to evaluate the script:

1. evaluate the banner output and decide UP/DOWN
2. evaluate the exit status of the script 0 - UP / !=0 DOWN

Would be nice if we have an example for both variants.

**Comment by mfuhrmann** [comment]
@Bonrob2 I added the exit status example as @indigo423 suggested and removed the note because I think with those 2 examples it should be self-explaining.

@indigo423 If no banner parameter is set, the exit status will be used, right?

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/SystemExecuteMonitor.adoc [hunk 1]
```diff
@@ -33,6 +33,8 @@ If the banner is not available in the output the status is determined as down.
 
 The parameter `args` supports variable replacement for the following set of variables.
 
+NOTE: It is recommended to set reasonable output before the script exists, because the output will be added into the event reason.
```
[the remark this answers is not in the document]
Comment (mfuhrmann) [reply opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/SystemExecuteMonitor.adoc:36]: But now that I reread it, I think we should give that hint. But where? In the the table of the banner parameter description? As note as above or maybe in the example?

**Comment by mfuhrmann** [comment]
@Bonrob2 Can you please review again?
@indigo423 I've added an other example script.

File: opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/SystemExecuteMonitor.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (Bonrob2) [reply opennms-doc/guide-admin/src/asciidoc/text/service-assurance/monitors/SystemExecuteMonitor.adoc:36]: I think the note is fine here, but just needs to be clarified. "what is reasonable output" and they are setting it before they create the script or before it runs?

**Comment by mfuhrmann** [comment]
Thanks for taking care! @indigo423

**Pull request merged**